config/monitoring/prometheus/base/prometheus-cm-service-ca.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: prometheus-service-ca
+  namespace: redhat-ods-monitoring
```

Comment:
these might need to transfer to template, for the namespace

Reply:
See comment above